adopters/cncf/README.md

```diff
@@ -0,0 +1,42 @@
+# Company: CNCF, The Linux Foundation
+
+# Stateful Applications that we are running on OpenEBS
+
+CNCF:
+- PostgresSQL (including automatic backups)
+- NFS server (for allowing multiple r/w access)
+- nginx (serving home pages, backups, CSV reports etc)
+- Storage for git repositories clones
+- All DevStats applications (homegrown)
+
+The Linux Foundation:
+- MariaDB database (Helm master + slave replication, automatic backups)
+- ElasticSearch cluster (Helm chart master + 4 slave nodes)
+- Postgres (including automatic backups)
+- Redis
+- Grimoire stack (CHAOSS)
```

Comment:
should we provide one more link here or below: https://chaoss.community/

Reply:
Those are for the Linux Foundation, list for CNCF is different. My comment had CNCF and LF cases separated.

Reply:
Ok, thanks @lukaszgryglicki